contrib/IECoreAppleseed/src/IECoreAppleseed/ShaderNetworkAlgo.cpp

```diff
@@ -89,7 +89,11 @@ renderer::ShaderGroup *convert( const IECoreScene::ShaderNetwork *shaderNetwork
 				shaderType += 4;
 			}
 
-			asr::ParamArray params( ParameterAlgo::convertShaderParameters( shader->parameters() ) );
+			IECore::ConstCompoundDataPtr expandedParameters = IECoreScene::ShaderNetworkAlgo::expandSplineParameters(
+				shader->parametersData();
```

Comment:
Stray `;` is breaking the build. I don't expect this to work though (once you add tests), because `convertShaderParameters()` doesn't appear to support Color3fVectorData yet.